docs/content/docs.md

```diff
@@ -1475,14 +1475,14 @@ will only work well for the bucket based backends (eg s3, b2,
 azureblob, swift) but these are the only backends likely to have
 millions of entries in a directory.
 
-### --log-file=FILE
+### --log-file
 
-Log all of rclone's output to FILE.  This is not active by default.
+Log all of rclone's output to file.  This is not active by default.
```

Comment:
This no longer makes sense without the `FILE` parameter. Could be "Log all of rclone's output to **a** file"?

Same with all the other changes in this commit (The `FILE` removal commit)